CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.5.0
+
+* Add support for on-premises and hybrid environments. ([#12])
+* Symlinked `/var/lib/amazon/ssm` to `/.bottlerocket/host-containers/NAME/ssm` so that SSM Agent state data can persist between boots. ([#12])
```

Comment:
Does anyone have a preference on if this should be `/NAME/`, `/<name>/`, or `/${HOST_CONTAINER_NAME}/`?

Reply:
Isn't it going to be `current`?

Reply:
We're going to go with an environment variable instead. Check out https://github.com/bottlerocket-os/bottlerocket/pull/1422.

Reply:
We should probably use the real variable here, right?  It's still as clear when reading, and can actually be used.